src/telemetry.ts

```diff
@@ -17,6 +18,29 @@ export interface TelemetryProperties {
 
 const telemetryService: Promise<TelemetryService> = getTelemetryService('redhat.vscode-tekton-pipelines');
 
+export function telemetryProperties(commandId: string): TelemetryProperties {
+  return {
+    identifier: commandId,
+  }
+}
+
+export async function telemetryLogError(commandId: string, result: string | Error): Promise<void> {
+  if (commandId) {
+    const telemetryProps: TelemetryProperties = telemetryProperties(`${commandId}_error`);
+    let message = getStderrString(result);
+    const clusterInfo = /lookup\s+([^']+):/;
+    if (clusterInfo.test(message)) {
+      message = message.replace(clusterInfo, 'lookup cluster info: ');
+    }
+    const urlCheck = /(https?:\/\/[^\s]+)/g;
```

Comment:
you can extract `clusterInfo` and `urlCheck` out of this function to not create them on each call